### PR TITLE
Create `Resource` object

### PR DESCRIPTION
**Pull request by MattWong-ca**
Created `Resource` object, which will be used when users submit a new resource. 
- Also added `Organizer` object
- Wasn't sure about `Age` and `Date`, think we should discuss these ones a little more (not sure what their types would be). Update: `Age` will be String array, `Date` will be String

### CODE REVIEW

**Comment by MattWong-ca** [comment]
The `ResourceHandlerServlet` and `ResourceListServlet` are meant to be for when a user enters values into a form and it retrieves these and stores them in Datastore

**Review by DPWXY — COMMENTED** [review]
I notice you comment out the datastore part for now. The Resource object looks good for me, I am not so sure whether we should merge this commented out file to the main.

File: project/src/main/java/com/google/sps/servlets/ResourceHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        // String[] ethnicity;
+
+        // Print the value so you can see it in the server logs.
+        System.out.println("You submitted: " + organizerName);
+
+        Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+        KeyFactory keyFactory = datastore.newKeyFactory().setKind("Resource");
+        FullEntity taskEntity = Entity.newBuilder(keyFactory.newKey())
+                .set("organizerName", organizerName)
+                // need to add other ones too, will be able to verify once frontend has JS ready
```
Comment (MattWong-ca) [inline project/src/main/java/com/google/sps/servlets/ResourceHandlerServlet.java]: @ericachen1 on line 40 we'll need to add in the right parameter names, won't be able to check the datastore table until frontend's JS is ready though (edit: actually nvm, we should be able to)

**Comment by MattWong-ca** [comment]
> I notice you comment out the datastore part for now. The Resource object looks good for me, I am not so sure whether we should merge this commented out file to the main.

Just uncommented it, I'm not sure how to take in user inputs for the checkmark options (eg. target age), so I left those string arrays commented for now (in the Resource.java file).

**Comment by MattWong-ca** [comment]
Otherwise @DPWXY @ericachen1 does everything else look good to merge in?

**Pull request merged**